README.md

```diff
@@ -74,18 +75,67 @@ For Fedora/RHEL clusters, the following might be required:
+For all of the following commands, set the `KUBECONFIG` env variables as instructed by `local-up-cluster.sh` or as needed for some other cluster.
+
+`deploy/kubernetes/deploy.sh` will deploy the nfs driver using an
+image from quay.io which (at the time of writing this) isn't available
+yet.
+
+It is possible to use a locally built image without any registry:
+``` sh
+$ make container
+...
+Successfully tagged nfsplugin:latest
+
+$ NFSPLUGIN_REGISTRY=none NFSPLUGIN_TAG=latest deploy/kubernetes/deploy.sh
+applying RBAC rules
+kubectl apply -f https://raw.githubusercontent.com/kubernetes-csi/external-attacher/v1.0.1/deploy/kubernetes/rbac.yaml
+serviceaccount/csi-attacher created
+clusterrole.rbac.authorization.k8s.io/external-attacher-runner created
+clusterrolebinding.rbac.authorization.k8s.io/csi-attacher-role created
+role.rbac.authorization.k8s.io/external-attacher-cfg created
+rolebinding.rbac.authorization.k8s.io/csi-attacher-role-cfg created
+deploying nfs plugin components
+   deploy/kubernetes/csi-attacher-nfsplugin.yaml
+        using           image: quay.io/k8scsi/csi-attacher:v1.0.1
+        using           image: nfsplugin:latest
+service/csi-attacher-nfsplugin created
+statefulset.apps/csi-attacher-nfsplugin created
+   deploy/kubernetes/csi-nodeplugin-nfsplugin.yaml
+        using           image: quay.io/k8scsi/csi-node-driver-registrar:v1.0.2
+        using           image: nfsplugin:latest
+daemonset.apps/csi-nodeplugin-nfsplugin created
+   deploy/kubernetes/csi-nodeplugin-rbac.yaml
+serviceaccount/csi-nodeplugin created
+clusterrole.rbac.authorization.k8s.io/csi-nodeplugin created
+clusterrolebinding.rbac.authorization.k8s.io/csi-nodeplugin created
+10:53:11 waiting for nfs deployment to complete, attempt #0
+10:53:21 waiting for nfs deployment to complete, attempt #1
+```
 
-If you already have a driver installed, skip to step 4 of the following steps.
+Other clusters may need a registry to pull from:
+``` sh
+$ make push REGISTRY_NAME=my-registry:5000
+...
+$ NFSPLUGIN_REGISTRY=my-registry:5000 NFSPLUGIN_TAG=latest deploy/kubernetes/deploy.sh
+```
 
-1) Build the nfs driver by running `make`
-2) Create NFS Driver Image, where the image tag would be whatever that is required by your YAML deployment files        `docker build -t quay.io/k8scsi/nfsplugin:v1.0.0 .`
-3) Install the Driver: `kubectl create -f deploy/kubernetes`
-4) Build E2E test binary: `make build-tests`
-5) Run E2E Tests using the following command: `./bin/tests --ginkgo.v --ginkgo.progress --kubeconfig=/var/run/kubernetes/admin.kubeconfig`
 
+Once you have the driver installed, tests can be run with:
+``` sh
+$ make build-tests
+mkdir -p bin
+CGO_ENABLED=0 GOOS=linux go build -a -ldflags '-X main.version=4fa924a251193c9eef937042112462433089d658 -extldflags "-static"' -o ./bin/tests ./cmd/tests
+$ ./bin/tests --ginkgo.v --ginkgo.progress
```

Comment:
do you know where the kubeconfig gets passed?

Reply:
`KUBECONFIG` gets set by `start_cluster` based on the output of `kind get kubeconfig-path --name=csi-prow`.

It would be useful to make that more visible in the `prow.sh` output for those who try to reproduce what it does.

Reply:
I've added two more commits which show KUBECONFIG (needs to be two separate commits because they'll go into different repos).